src/QMCDrivers/tests/ValidQMCInputSections.h

```diff
@@ -0,0 +1,79 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2019 QMCPACK developers.
+//
+// File developed by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+//
+// File created by: Peter Doak, doakpw@ornl.gov, Oak Ridge National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#ifndef QMCPLUSPLUS_VALIDQMCINPUTSECTIONS_H
+#define QMCPLUSPLUS_VALIDQMCINPUTSECTIONS_H
+
+/** @file
+ *  @brief avoids many repeated xml heredoc sections
+ */
+
+#include <array>
+
+namespace qmcplusplus
+{
+namespace testing
+{
+// clang-format: off
+constexpr std::array<const char*, 2> valid_vmc_input_sections{
+    R"(
+  <qmc method="vmc" move="pbyp">
+    <estimator name="LocalEnergy" hdf5="no" />
+    <parameter name="walkers">                1 </parameter>
+    <parameter name="stepsbetweensamples">    1 </parameter>
+    <parameter name="warmupSteps">            5 </parameter>
+    <parameter name="substeps">               5 </parameter>
+    <parameter name="steps">                  1 </parameter>
+    <parameter name="blocks">                 2 </parameter>
+    <parameter name="timestep">             1.0 </parameter>
+    <parameter name="usedrift">              no </parameter>
+  </qmc>
+)",
+    R"(
+  <qmc method="vmc_batch" move="pbyp">
+    <parameter name="crowds">                 8 </parameter>
+    <estimator name="LocalEnergy" hdf5="no" />
+    <parameter name="walkers">                32 </parameter>
+    <parameter name="stepsbetweensamples">    1 </parameter>
```

Comment:
I'd like to retire this input parameter.

Reply:
Agreed. Worth noting, but perhaps this is not the place to fix. Additionally it is a duplication of the already badly named substeps.